modules/data-loading/pages/data-streaming-connector/aws-s3.adoc

```diff
@@ -59,6 +68,84 @@ You can adopt them in your connector config files with prefix `file.reader.setti
 
 include::partial$parsing-rules.adoc[]
 
+include::partial$map-source-to-connector.adoc[]
+
+==== Example
+
+The following is an example configuration file:
+
+----
+connector.class=com.tigergraph.kafka.connect.filesystem.aws.S3SourceConnector
+file.reader.settings.fs.s3a.aws.credentials.provider=org.apache.hadoop.fs.s3a.SimpleAWSCredentialsProvider
+file.reader.settings.fs.s3a.access.key=A2V************J
+file.reader.settings.fs.s3a.secret.key=wEV************************************8
+file.reader.settings.fs.s3a.impl=org.apache.hadoop.fs.s3a.S3AFileSystem
+file.reader.settings.fs.AbstractFileSystem.s3a.impl=org.apache.hadoop.fs.s3a.S3A
+file.reader.settings.fs.s3a.threads.max=1000
+file.reader.settings.fs.s3a.connection.maximum=1000
+
+mode=eof
+file.regexp=".*"
+file.recursive=true
+file.reader.type=text
+file.reader.batch.size=10000
+file.reader.text.eol="\\n"
+file.reader.text.header=true
+file.reader.text.archive.type=auto
+file.reader.text.archive.extensions.tar=tar
+file.reader.text.archive.extensions.zip=zip
+file.reader.text.archive.extensions.gzip=tar.gz,tgz
```

Comment:
My bad. The config should be like
```
file.reader.text.archive.extensions.gzip=gz
file.reader.text.archive.extensions.tar.gz=tar.gz,tgz
```
due to the [recent change](https://github.com/tigergraph/cqrs/pull/1305) on connector. I updated the [user manual](https://graphsql.atlassian.net/wiki/spaces/infra/pages/2804973649/AWS+S3+Connector).